code/game/objects/items/weapons/scissors.dm

```diff
@@ -45,6 +48,12 @@
 			for(var/i in hair_styles_list)
 				var/datum/sprite_accessory/hair/tmp_hair = hair_styles_list[i]
 				if(H.species.name in tmp_hair.species_allowed)
+					if(H.species.name == "Machine")
+						if(!(organs_data["head"])) //If they have the default Morpheus head, you can't be messing with their display.
```

Comment:
this is literally a global list that is set every time a preference datum is transfered to someone
what the fuck